webui/react/src/types.ts

```diff
@@ -322,6 +324,11 @@ export interface ValidationMetrics {
   validationMetrics: Record<string, number>;
 }
 
+export interface TrialPagination {
```

Comment:
suggestion: I think pagination like startEndTimes is gonna repeat in all our endpoints that return a list exactly the same. wdy about sth like this

![2021-02-02_12-10-06](https://user-images.githubusercontent.com/12127420/106656586-bb0dd600-654f-11eb-8b7a-49425f47d38d.png)
~another thought if we define it this way maybe we don't define a new  Trials or TrialPagination type. anywhere we return a value that has pagination we define the return type as `T & Pagination`~

Reply:
great idea, I just did another one (active experiments) and thought the same thing